as/src/base/index.c

```diff
@@ -31,7 +31,23 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <string.h>
+
+#if defined(__x86_64__) || defined(__i386__) || defined(_MSC_VER)
 #include <xmmintrin.h>
+#elif defined(__powerpc64__)
+// cache prefetch transient
+static inline void
+_mm_prefetch(void *pointer, int unused __attribute__((unused)))
+{
+  __asm__ ( " dcbtt     0,%0;" : : "b" (pointer));
+}
+#else
+#define _mm_prefetch(X, Y) __builtin_prefetch(X)
```

Comment:
_MM_HINT_NTA implies that it is a non-temporarl fetch, therefore `0` can be used as the second argument to __builtin_prefetch. Could check the ISA2.07 to see about dcbtt and see if it also is a non-temporal fetch? Maybe the __builtin_prefetch is sufficient on POWER too.

Reply:
will you make the recommendation on this? this change was part of your original commit:

https://github.com/grooverdan/aerospike-server/commit/e07e3744b94d9da862c663fc479030d48874a213